### DISCUSSION TITLE
Is it possible to change the display name of a team?

### DISCUSSION THREAD

**Discussion by bmiller52** [MLB-LED-Scoreboard/mlb-led-scoreboard#554] [Q&A] [answered]
I have been enjoying my 64x32 scoreboard all summer and have customized it to my liking.  I prefer to have the full name of the teams displayed opposed to just the abbreviations, but as already known if the hit and run columns are double digits, they overlap with the team name.  But only a few names actually are long enough to overlap in that scenario ie the Nationals, Athletics, Cardinals among others.  I would like to use the common nicknames for those teams ie the Nats, A's, Cards etc.  

In the `data/teams.py` file, I was hoping to edit the values in `TEAM_ABBR_LN` with the nicknames, and specify `full_team_names` to `false` in the `config.json` file.  But that didn't change anything.  Does anyone know a way to edit the display team names?

Btw thanks to all that have worked on this project, such cool stuff!

**Comment by c2mfj** [comment]
Hi Brett, I don't know the answer but had the same thought! The answer I
was given is in the discord.

https://discord.com/channels/1091751372069806183/1091763502378651768/1149681282264092734

[image: image.png]

**Reply by bmiller52** [reply]
Unfortunately I don't have access to that discord channel.  What did they suggest?

**Comment by c2mfj** [comment, marked as answer]
The shortnames that you (and I tried) altering are specifically for the
playoff brackets in the post-season. Some customization to code would be
required to utilize custom shortnames in the game scoreboard.

**Reply by bmiller52** [reply]
Ok good to know
